src/background.js

```diff
@@ -1,6 +1,6 @@
 const REDIRECT_URL = browser.identity.getRedirectURL();
 const CLIENT_ID = 'c6d74070a481bc10';
-const SCOPES = ['profile keys'];
+const SCOPES = ['kinto profile keys'];
```

Comment:
The scope just need to be in sync with the kinto server configuration because it will verify that the scope is correct. For kinto.dev the mandatory scope is kinto :)